doc/rst/source/supplements/geodesy/earthtide.rst

```diff
@@ -95,6 +95,7 @@ Optional Arguments
     current time in UTC from the computer clock. If no **-G** or **-S** are provided then **-T** is interpreted to mean compute
     a time-series at the location specified by **-L**, thus then **-L** becomes mandatory.
     When **-G** and **-T**, only first time T series is considered. Finally, dates may range from 1901 through 2099.
+    For details on array creation, see `Generate 1D Array`_.
```

Comment:
You need to add `.. include:: /explain_array.rst_`

Reply:
Thanks that worked.